Menu.js

```diff
@@ -25,9 +23,32 @@ module.exports.getMenuTemplate = () => {
     },
     {
       role: 'window',
+      submenu: [{ role: 'minimize' }, { role: 'close' }]
+    },
+    {
+      label: 'Config',
+      submenu: [
+        {
+          label: 'edit',
+          click() {
+            startConfigEditor()
+          }
+        }
+      ]
+    },
+    {
+      label: 'Plugins',
       submenu: [
-        { role: 'minimize' },
-        { role: 'close' }
+        {
+          label: 'rescan',
+          click() {
+            if (!global.PluginHost) {
+              console.log('plugin host not ready')
+              return
+            }
+            global.PluginHost.loadUserRegistries()
```

Comment:
this is broken but should be fixed with #296

Reply:
@ryanio see ^